src/main/java/org/openstreetmap/atlas/checks/validation/intersections/EdgeCrossingEdgeCheck.java

```diff
@@ -178,6 +199,40 @@ public boolean validCheckForObject(final AtlasObject object)
         return Optional.of(newFlag);
     }
 
+    /**
+     * Check if {@link Edge} highway tag is vehicle or pedestrian navigable.
+     *
+     * @param edge
+     *            Atlas object.
+     * @param carNav
+     *            Car Navigable Highway Type.
+     * @param pedestrianNav
+     *            Pedestrian Navigable Highway Type.
+     * @return {@code true} if {@link Edge} is vehicle or pedestrian navigable.
+     */
+    private boolean getCrossingHighwayType(final Edge edge, final boolean carNav,
+            final boolean pedestrianNav)
```

Comment:
Please don't use abbreviations in variable names. And as this is return a boolean isCrossingHighwayType would make more sense.